### PR TITLE
[TIMOB-20591] iOS: Rename "tableSeparatorInsets" to "listSeparatorInsets" in Ti.UI.ListView

### PR DESCRIPTION
**Pull request by hansemannn**
**JIRA:** https://jira.appcelerator.org/browse/TIMOB-20591

### CODE REVIEW

File: iphone/Classes/TiUIListView.m [hunk 1]
```diff
@@ -546,10 +546,17 @@ -(NSInteger)sectionForSearchSection:(NSInteger)section
 -(void)setSeparatorInsets_:(id)arg
 {
     [self tableView];
-    DEPRECATED_REPLACED(@"UI.ListView.separatorInsets", @"5.2.0", @"UI.ListView.tableSeparatorInsets");
+    DEPRECATED_REPLACED(@"UI.ListView.separatorInsets", @"5.2.0", @"UI.ListView.listSeparatorInsets");
 }
+
 -(void)setTableSeparatorInsets_:(id)arg
```
Comment (AngelkPetkov) [inline iphone/Classes/TiUIListView.m]: change to setListSeparatorInsets

Comment (hansemannn) [inline iphone/Classes/TiUIListView.m]: No, the method needs to stay there to display the deprecation message

Comment (AngelkPetkov) [inline iphone/Classes/TiUIListView.m]: Oh yeah didn't see you added setListSeparator. My bad

**Comment by AngelkPetkov** [comment]
FT and CR approved!

**Pull request merged**